Untitled.ipynb

```diff
@@ -0,0 +1,561 @@
+{
```

Comment:
Hmm, maybe notebooks shouldn't be added to the repo (and in fact even added to .gitignore?)